data/html-parser/index.ts

```diff
@@ -0,0 +1,83 @@
+// Essentially jQuery for parsing the DOM
+import cheerio from 'cheerio';
+import { defaults, pickAll } from 'lodash/fp';
+import DataTypes from '../types';
+import HtmlDataTypes from '../types/html';
+import { duplicateLangAttributes } from './duplicate-lang-attributes';
+import { liftLangAttributes } from './lift-lang-attributes';
+import { removeParagraphsFromDefinitionListsAndMerge } from './remove-paragraphs-from-definition-lists';
+
+type Attributes = { [attr: string]: string };
+
+type HtmlDataType = keyof typeof HtmlDataTypes;
+
+/**
+ * Data that we need to check from any cheerio element to see what to do with it
+ */
+type AnyCheerioElement = {
+  type?: null | undefined | 'text' | 'tag' | 'script' | 'style' | 'comment';
```

Comment:
Don't need question mark + undefined, remove undefined from this list (but not null)

Reply:
Cool! Removed.